src/db/db_place.h

```diff
@@ -47,12 +47,13 @@ namespace db{
             void clear(){
                 _placement.clear();
             }
-            int x(Cell *cell) const {
+            int x(Cell *cell) {
                 return _placement.at(cell).x();
-                //return _placement[cell].x();
+                // return _placement[cell].x();
             }
             int y(Cell *cell) {
-                return _placement[cell].y();
+                return _placement.at(cell).y();
+                // return _placement[cell].y();
             }
```

Comment:
Using the `at` function doesn't create new entries if the indices don't exist.
I think it would be safe to declare these two functions as `const`.

Reply:
Sure.